tests/Linter/ProcessLinterTest.php

```diff
@@ -37,9 +37,9 @@ public function testIsAsync()
      *
      * @testWith ["php", "foo.php", "'php' '-l' 'foo.php'"]
      *           ["C:\\Program Files\\php\\php.exe", "foo bar\\baz.php", "'C:\\Program Files\\php\\php.exe' '-l' 'foo bar\\baz.php'"]
-     * @requires OS Linux
+     * @requires OS Linux|Darwin
      */
-    public function testPrepareCommandOnPhpOnLinux($executable, $file, $expected)
+    public function testPrepareCommandOnPhpOnLinuxAndMac($executable, $file, $expected)
```

Comment:
@keradus shall I split this off into a separate test?

In general, cases that pass on Linux should also on Mac without modification (unless of course the OS name parsing is causing bugs 😉 ).